packages/experiments/src/components/Card/examples/Card.Basic.Example.tsx

```diff
@@ -0,0 +1,57 @@
+// @codepen
+import * as React from 'react';
+import { Card } from '../Card';
+import { Persona } from '../../../Persona';
+import { PersonaTestImages } from '@uifabric/experiments/lib/common/TestImages';
+import { Stack } from '../../../Stack';
+import { Text } from '../../../Text';
+import { Icon, Image } from 'office-ui-fabric-react';
+import { mergeStyleSets } from 'office-ui-fabric-react/lib/Styling';
+
+export class CardBasicExample extends React.Component<{}, {}> {
+  public render(): JSX.Element {
+    const styles = mergeStyleSets({
+      textLarge: {
+        color: 'teal',
+        fontWeight: 'bolder'
+      },
+      textBold: {
+        fontWeight: 'bold'
+      }
+    });
+
+    return (
+      <Stack gap={5}>
```

Comment:
Is this Stack needed?

Reply:
Good catch! I initially had it in case we had more than one example, but given that I'm only adding one at this time I'll remove the `Stack`.